.github/workflows/test_client.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           [source.crates-io]
           replace-with = "vendored-sources"
           [source.vendored-sources]
-          directory = "$PWD/vendor"
+          directory = "$PWD/rust-vendor"
```

Comment:
Not being a rust expert, I'm unsure why this change is made.

Reply:
The ubuntu-mir Rust policy wants this folder to be called `rust-vendor`. See https://github.com/canonical/ubuntu-mir/blob/main/vendoring/Rust.md#automation-via-debianrules